apps/asap-server/src/entities/news-and-events.ts

```diff
@@ -12,7 +12,9 @@ export const parseNewsAndEvents = (
     text: item.data.text?.iv,
     link: item.data.link?.iv,
     linkText: item.data.linkText?.iv,
-    thumbnail: item.data.thumbnail && createURL(item.data.thumbnail?.iv)[0],
+    thumbnail:
+      (item.data.thumbnail?.iv && createURL(item.data.thumbnail.iv)[0]) ||
```

Comment:
Maybe this could be a ternary expression. Do you think it is worth the change?

Reply:
sure, done